tox.ini

```diff
@@ -48,7 +48,7 @@ commands =
     {toxinidir}/run-tests.sh {posargs}
     {toxinidir}/run-func-tests.sh {posargs}
 
-[testenv:py35-postgresql-file-upgrade-from-3.1]
+[testenv:py36-postgresql-file-upgrade-from-3.1]
```

Comment:
i should ask why you change to 3.6 right? i'll ask why... why you change to 3.6 if everything else is 3.5? :)